redirect/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Redirecting to https://radapp.io</title>
+    <meta http-equiv="refresh" content="0; URL=https://radapp.io">
```

Comment:
Checked and this uses an HTTP 301 which is correct.

Reply:
Will this handle both `radapp.dev` and `docs.radapp.dev`?

Reply:
Nope that'll need to be tackled by a different mechanism that retains the path in the URL. Probably a web.config file that updates the app service configuration. Can do that as a follow-up to this PR